cypress_test/integration_tests/desktop/impress/top_toolbar_spec.js

```diff
@@ -29,6 +29,66 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'font-weight', '700');
 	});
 
+	it('Apply italic on text shape.', function() {
+		cy.get('#tb_editbar_item_italic')
+			.click();
+
+		impressHelper.triggerNewSVGForShapeInTheCenter();
+
+		cy.get('.leaflet-pane.leaflet-overlay-pane g.Page .TextParagraph')
+			.should('have.attr', 'font-weight', '400');
+	});
```

Comment:
In the beginning, it would be better if you can focus on one test case only. So we can go on step by step. Let's focus on the italic font attribute first.

This issue here is with the assertion at the end of the test case. Checking font-weight is good when we are testing bold fonts, but applying italic won't change font-weight. You should check the mobile test, how to test whether the italic attribute was applied on the shape:
https://github.com/CollaboraOnline/online/blob/master/cypress_test/integration_tests/mobile/impress/apply_font_shape_spec.js